db-ops/create-db-user.sh

```diff
@@ -13,6 +13,7 @@ DB_APP_USERNAME=$DB_NAME
 DB_APP_PASSWORD=$(LC_ALL=C tr -dc 'A-Za-z0-9' < /dev/urandom | base64 | head -c 16)
 
 # Fill in env-vars to db user creation manifest
+JOB_ID=$(LC_ALL=C tr -dc 'a-z0-9' < /dev/urandom | head -c 8)
```

Comment:
line 26 will need `-$JOB_ID` as well

Reply:
or actually the original goal was to delete the `db-ops` namespace on `Line 29`(instead of `get`) 
 but when I was troubleshooting I changed it and forgot about it, lets change it back to deleting it? (since the job pod will have the rds-master password)

Reply:
- add $JOB_ID for other places.
- delete namespace